src/components/__stories__/artwork_filter.tsx

```diff
@@ -0,0 +1,144 @@
+import { action, storiesOf } from "@kadira/storybook"
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import { artsyNetworkLayer } from "../../relay/config"
+import { FilterArtworksQueryConfig } from "../../relay/root_queries"
+
+import Dropdown from "../artwork_filter/dropdown"
+import TotalCount from "../artwork_filter/total_count"
+
+interface FilterArtworksDropdownState {
```

Comment:
@broskoski Is there a reason why you put `FilterArtworksDropdown` and its state interface in a story file?